src/rez/rezconfig.py

```diff
@@ -657,6 +657,11 @@
 # - "override": Package's preprocess function completely overrides the global preprocess.
 package_preprocess_mode = "override"
 
+# Defines whether a resolve should immediately fail if any variants have a package that can't be found.
```

Comment:
This is confusing. Variants don't have packages... The hierarchy is package family (parent of all versions) > package (version) > variant.

Reply:
What term would you use to refer to a package requested by a variant? I can change that accordingly, but I think a variant refers to the entire list e.g `['foo', 'spam', 'eggs']` is one variant, but this applies if one of those items in the list are missing.

I can change it around to new wording, but I think calling it `variants` is also perhaps not correct? Maybe `variant requires` ?